lib/templates/src/components/Chat.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <div id="chat-box">
+      <p v-for="message in messages" :key="message" v-text="message"></p>
+    </div>
+    <div class="magpie-view-answer-container">
+      <textarea
+        ref="text"
+        cols="50"
+        placeholder="Type your message to the other participant here."
+      ></textarea>
+      <button @click.stop="send()">
+        Send
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+const EVENT_CHAT_MESSAGE = 'chat_message';
+
+export default {
+  name: 'Chat',
+  data() {
+    return {
+      messages: []
+    };
+  },
+  socket: {
+    [EVENT_CHAT_MESSAGE](payload) {
+      this.messages.push(payload);
+    }
+  },
+  EVENT_CHAT_MESSAGE,
```

Comment:
What's the purpose of this line, actually?

Reply:
Ah, I just added this as a convenience for users of the component that want to do something additional with the broadcast event.